widget/embedded/src/hooks/useTheme.ts

```diff
@@ -1,26 +1,35 @@
 import type { WidgetTheme } from '../types';
 
-import { createTheme, darkTheme, lightTheme } from '@rango-dev/ui';
+import {
+  createTheme,
+  darkTheme,
+  lightTheme,
+  darkColors as mainDarkColors,
+  theme as mainTheme,
+} from '@rango-dev/ui';
 import { useEffect, useState } from 'react';
 
 import { useMetaStore } from '../store/meta';
 import { useSettingsStore } from '../store/settings';
+import { generateColors } from '../utils/common';
 import {
   DEFAULT_PRIMARY_RADIUS,
   DEFAULT_SECONDARY_RADIUS,
 } from '../utils/configs';
 
-export function useTheme({
-  // colors,
-  fontFamily = 'Roboto',
-  borderRadius = DEFAULT_PRIMARY_RADIUS,
-  secondaryBorderRadius = DEFAULT_SECONDARY_RADIUS,
-  mode = 'auto',
-}: WidgetTheme) {
+export function useTheme(props: WidgetTheme) {
+  const {
+    colors,
+    fontFamily = 'Roboto',
```

Comment:
Please use a constant variable for here like DEFAULT_FONT_FAMILY.

Reply:
fixed